.github/workflows/lint.yml

```diff
@@ -24,5 +24,8 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Compile v3 contracts
+        run: make compile-v3-contracts
+
```

Comment:
Why do we need this in the linter workflow?

Reply:
Because of this `go:embed` (build and linting fails if we don't compile smart contracts):
https://github.com/0xPolygon/polygon-edge/pull/774/files#diff-30098e78e2b1b570e6f9b69a37aac503d2c67c8088d00b95e45ae54b90b49268R12

This go:embed should be removed once we integrate with the v3-contracts and remove these dummy ones.